src/main/java/com/ombremoon/enderring/common/object/world/effect/StatusEffect.java

```diff
@@ -52,6 +52,9 @@ public void applyEffectTick(LivingEntity pLivingEntity, int pAmplifier) {
         } else if (this == StatusEffectInit.BLESSED_DEW_TALISMAN.get()) {
             pLivingEntity.heal(0.13F);
         }
+        else if(this == StatusEffectInit.SPECKLEDHARD_CRYSTAL.get()&& pLivingEntity instanceof Player player){
+            player.removeAllEffects();
```

Comment:
this line is also gonna remove effects from currently equipped talismans and any other tear effects active. Needs modifying to only remove things like build up effects.

Reply:
alright